api/binance/futures_client.py

```
@@ -1,901 +1,901 @@
+
+        except requests.exceptions.HTTPError as e:
+            error_msg = e.response.text if e.response else 'N/A'
+            error_code = -9999  # 기본값
+
+            # 에러 정보 파싱
+            if e.response:
+                try:
+                    error_json = e.response.json()
+                    error_code = error_json.get("code", -9999)
+                    error_msg = error_json.get("msg", str(e))
+                    logger.error(f"API 오류 상세: {error_json}")
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)